src/katello/utils.py

```diff
@@ -7,7 +7,7 @@
     from ConfigParser import ConfigParser
 
 def plugin_enabled(filepath, environment_variable, force = False):
-    return force or (config_enabled(filepath) and not environment_disabled(environment_variable ))
+    return (force or (config_enabled(filepath) and not environment_disabled(environment_variable))) and not subman_profile_enabled()
```

Comment:
The way I've written this - force does *not* force if we have subman combined profiles. Easily changed if we see any value in it